google-map.html

```diff
@@ -511,217 +147,295 @@
     }
 
   </style>
+  <template>
+    <google-maps-api 
+      api-key="{{apiKey}}" 
+      client-id="{{clientId}}" 
+      version="{{version}}" 
+      libraries="{{libraries}}" 
```

Comment:
They can, I'll do it. The prevailing Polymer style right now is not to be too strict about this.